_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+| --------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {% include custom-anchor name="error.connector.http.methodNotAllowed" %}                                              | The request method is not supported for the requested resource.                                                                                                           |
+| {% include custom-anchor name="error.connector.http.notAcceptable" %}                                                 | The requested resource is capable of generating only content not acceptable according to the Accept headers sent in the request.                                          |
+| {% include custom-anchor name="error.connector.routeDoesNotExist" %}                                                  | This route does not exist.                                                                                                                                                |
+| {% include custom-anchor name="error.connector.unauthorized" %}                                                       | Unauthorized.                                                                                                                                                             |
+| {% include custom-anchor name="error.connector.validation.invalidJsonInPayload" %}                                    | The given payload is not a valid json object.                                                                                                                             |
+| {% include custom-anchor name="error.consumption.attributes.cannotSucceedAttributesWithAParent" %}                    | The Attribute you want to succeed has a parent. You cannot succeed Attributes with a parent. Instead, succeed the parent, which will implicitly succeed all its children. |
+| {% include custom-anchor name="error.consumption.attributes.predecessorNotFound" %}                                   | An Attribute with the given id does not exist. Please use create instead.                                                                                                 |
+| {% include custom-anchor name="error.consumption.requests.decide.validation.invalidNumberOfItems" %}                  | The request and the response item lengths are different.                                                                                                                  |
+| {% include custom-anchor name="error.consumption.requests.decide.validation.invalidRequestId" %}                      | The response is invalid because the id of the Request does not match the id of the Response.                                                                              |
```

Comment:
Geht leider nicht, ich kann nicht einfach nen Fehler werfen. Ich gebe an der stelle ValidationResult's zurück.

Reply:
`throw new Error("The response is invalid because the id of the Request does not match the id of the Response.")` geht doch immer. Der muss ja nicht abgefangen werden oder so, es geht ja nur drum, dass, sollte tatsächlich mal was falsches reingegeben werden, man eine anständige Fehlermeldung bekommt.

Reply:
Kommentarlos resolved, aber nichts geändert, wenn ich das richtig sehe, oder?

Reply:
Kommentarlos resolved, aber nichts geändert, wenn ich das richtig sehe, oder?